store/sqlstore/post_store.go

```diff
@@ -1043,7 +1043,7 @@ func (s *SqlPostStore) Search(teamId string, userId string, params *model.Search
 
 	_, err := s.GetSearchReplica().Select(&posts, searchQuery, queryParams)
 	if err != nil {
-		mlog.Warn(fmt.Sprintf("Query error searching posts: %v", err.Error()))
+		mlog.Warn("Query error searching posts:", mlog.Err(err))
```

Comment:
Probably here we can replace the `:` at the end with a `.`.

Reply:
done